build_and_publish_docker_images.sh

```diff
@@ -0,0 +1,2 @@
+docker-compose -f docker-compose-build.yaml --no-cache --parallel
```

Comment:
@chriseth would something like this be okay for building and pushing of docker images? 
Basically you have to be logged in locally to be able to publish it to docker hub.